examples/tls_server-rs/ta/src/main.rs

```diff
@@ -92,12 +94,23 @@ fn invoke_command(cmd_id: u32, params: &mut Parameters) -> Result<()> {
 }
 
 pub fn new_tls_session(session_id: u32) {
```

Comment:
1. This function should return a Result, as there are multiple failures inside its impl.
2. The unwrap should always be eliminated, in our codebase.
I suggest to run `cargo clippy` to help check and follow best Rust coding practices.